frontend/src/pages/projects/screens/detail/pipelines/PipelinesSection.tsx

```diff
@@ -23,6 +25,11 @@ const PipelinesSection: React.FC = () => {
           key={`action-${ProjectSectionID.PIPELINES}`}
           variant="secondary"
         />,
+        <PipelinesPageHeaderActions
+          key={`action-${ProjectSectionID.PIPELINES}`}
```

Comment:
I'm aware that the `key` should be unique. Any naming conventions that we follow?